src/components/bcros/filing/item/StaffFiling.vue

```diff
@@ -9,7 +9,10 @@
     </template>
 
     <template #subtitle>
-      <span v-if="putBackOnOrAdminDissolution">{{ $t('text.filing.filed') }}</span>
+      <template v-if="putBackOnOrAdminDissolution">
+        <span v-if="isStatusCompleted">{{ $t('text.filing.filed') }}</span>
+        <span v-else>{{ $t('text.filing.filedAndPending') }}</span>
+      </template>
```

Comment:
I can't show the "filed and pending" version because that requires the PBO to be "hung" (not completed).

However, here is a completed PBO (same as before):

![image](https://github.com/user-attachments/assets/c5feb55e-10ac-438b-b909-c5e7a3eb0fd5)